1.run_multi_IC_pipelines/scripts/multi_pipelines.py

```diff
@@ -52,16 +44,16 @@
 # create plate info dictionary with all parts of the CellProfiler CLI command to run in parallel
 plate_info_dictionary = {
     name: {    
-        "path_to_images": pathlib.Path(f"../data/test_plate/Plate_1/"),
+        "path_to_images": pathlib.Path(f"../data/test_plate/Plate_3_prime_sub/"),
         "path_to_output": pathlib.Path(f"{output_dir}/{name}_IC"),
         }
-        for name in plate_run_names
+        for name in pipeline_names
 }
 # iterate over the dictionary and add the path_to_pipeline 
 for name, info in plate_info_dictionary.items():
-        for i in range(len(plate_run_names)):
-            if name == "Plate_1_" + pipeline_names[i]:
-                info["path_to_pipeline"] = pathlib.Path(f"../pipelines/" + pipeline_names[i]+ ".cppipe")
+        for i in range(len(pipeline_names)):
+                if name == pipeline_names[i]:
+                    info["path_to_pipeline"] = pathlib.Path(f"../pipelines/" + pipeline_names[i]+ ".cppipe")
```

Comment:
```suggestion
        for i in range(len(pipeline_names)):
                if name == pipeline_names[i]:
                    info["path_to_pipeline"] = pathlib.Path(f"../pipelines/{pipeline_names[i]}.cppipe")
```
[f-strings](https://realpython.com/python-f-strings/)  in python are really cool - consider using them more the future for this concept called string interpolation!